spec/models/plugins/shared_examples.rb

```diff
@@ -37,6 +37,13 @@
     expect( subject.liquid_data({form_values: {email: 'neal@test.com'}})[:outstanding_fields]).to eq []
   end
 
+  it 'deletes the form when it is deleted' do
+    subject.form = form
+    subject.save!
+    expect{ subject.destroy! }.not_to raise_error
```

Comment:
To reproduce - try changing the layout on [this page on production](http://champaign-production.elasticbeanstalk.com/pages/abbott-government-offer-amnesty-to-foreign-workers-exploited-by-7-eleven/edit). I just did it, and we got [this 500 error](https://papertrailapp.com/groups/2166113/events?q=program%3Aproduction.log&r=631934676764000290-631934727800291347)